server/routes/merchants.js

```diff
@@ -0,0 +1,120 @@
+const express = require('express')
+const { isLoggedIn } = require('../middlewares')
+const Price = require('../models/Price')
+const Merchant = require("../models/Merchant")
```

Comment:
Lets keep things consistent. Try to use single quotes.

Reply:
Got it, will try to stick to single quotes from here on out.  Thanks for the input!